server_http.hpp

```diff
@@ -80,6 +80,7 @@ namespace SimpleWeb {
             std::string remote_endpoint_address;
             unsigned short remote_endpoint_port;
             
+            std::function<void(const std::exception&)> exception_handler;
```

Comment:
But I can do that later, your proposed change looks good. The problem with adding Request::address() and Request::port() is that the socket needs to be stored in the Request object somehow, and currently the Request objects can be stored longer than the socket object is alive.